package.json

```diff
@@ -29,6 +29,6 @@
     "repository": "git://github.com/GPII/linux.git",
     "main": "./gpii/index.js",
     "engines": {
-        "node": ">=4.2.1"
+        "node": ">=6.3.1"
```

Comment:
Does this pull request break compatibility with Node 4.x, or is our support now broadened to include both Node 4.x and 6.x? If it's the latter, we probably don't need to tighten up this Node.js engine version requirement.